source/material_model/reactive_fluid_transport.cc

```diff
@@ -246,12 +362,26 @@ namespace aspect
                              "computed. If the model does not use operator splitting, this parameter is not used. "
                              "Units: yr or s, depending on the ``Use years "
                              "in output instead of seconds'' parameter.");
+          prm.declare_entry ("Maximum weight percent water in sediment", "3",
+                             Patterns::Double (0),
+                             "The maximum allowed weight percent that the sediment composition can hold.");
+          prm.declare_entry ("Maximum weight percent water in MORB", "2",
+                             Patterns::Double (0),
+                             "The maximum allowed weight percent that the sediment composition can hold.");
+          prm.declare_entry ("Maximum weight percent water in gabbro", "1",
+                             Patterns::Double (0),
+                             "The maximum allowed weight percent that the sediment composition can hold.");
+          prm.declare_entry ("Maximum weight percent water in peridotite", "8",
+                             Patterns::Double (0),
+                             "The maximum allowed weight percent that the sediment composition can hold.");
```

Comment:
I have a question about these parameters. Is there a reason these are input parameters rather than part of the parametrization itself (like all the other parameters)? Or in other words: Are these related to physical properties of the rocks, or do they come from somewhere else (in which case, I would make a note of that).

Reply:
They come from somewhere else, the polynomial can reach very high values if it's unbounded at low P-T conditions, so this helps to keep the wt%'s reasonable! I made it variable because I figured a user might want to adjust this depending on what they think is going on in the system their measuring.